django/src/rdwatch/views/vector_tile.py

```diff
@@ -42,7 +20,7 @@ def vector_tile(request: HttpRequest, z: int, x: int, y: int):
     )
 
     evaluations_queryset = (
-        SiteEvaluation.objects.filter(intersects)
```

Comment:
Should we keep the filter for intersection as a secondary filter?
 I.E SiteEvaluation.objects.filter(configuration_id=hyper_parameters_id, intersects)`
We don't have many but if a model-run spans a large area it will cull some of polygons returned outside of the view.  I believe as long as the order is maintained like above it would do the configuration_id first and then the intersections

Reply:
Good point, I added this back in.